packages/react-mutation-mapper/src/component/oncokb/OncoKbCardTreatmentContent.tsx

```diff
@@ -0,0 +1,280 @@
+                                                    publicInstanceDisclaimerOverLay
+                                                }
+                                            >
+                                                <i
+                                                    className={
+                                                        'fa fa-info-circle'
+                                                    }
+                                                ></i>
+                                            </DefaultTooltip>
+                                        </p>
+                                    ) : (
+                                        <>
+                                            <p>{this.props.tumorTypeSummary}</p>
+
+                                            {this.props.treatments!.length >
+                                                0 && (
+                                                <div
+                                                    style={{
+                                                        marginTop: 10,
+                                                    }}
+                                                >
+                                                    <OncoKbTreatmentTable
+                                                        variant={
+                                                            this.props
+                                                                .variant || ''
+                                                        }
+                                                        pmidData={
+                                                            this.props.pmidData!
+                                                        }
+                                                        treatments={
+                                                            this.props
+                                                                .treatments!
+                                                        }
+                                                    />
+                                                </div>
+                                            )}
+                                        </>
+                                    )}
+                                </div>
+                            </div>
+                        )}
+                        {this.activeTab === ActiveTabEnum.MUTATION_EFFECT && (
+                            <div
+                                className={classnames(tabsStyles['tab-pane'])}
+                                data-test={`${ActiveTabEnum.MUTATION_EFFECT}-pane`}
+                                style={{
+                                    maxHeight: 200,
+                                    overflowY: 'auto',
+                                }}
+                            >
+                                {this.props.biologicalSummary !== undefined &&
```

Comment:
i find these nested ternary expressions to be too hard to read and very easy to mess up.  i would rather see <IF/Else> used here.